bundles/net.tourbook/src/net/tourbook/database/TourDatabase.java

```diff
@@ -7317,6 +7317,16 @@ private int updateDbDesign_039_to_040(final Connection conn, final SplashManager
             // Create index in table: TOURDATA_TOURTAG - Index: TOURTAG_TAGID
             SQL.CreateIndex(  stmt, JOINTABLE__TOURDATA__TOURTAG, KEY_TAG);
 
+// SET_FORMATTING_ON
+         }
+
+         if (isColumnAvailable(conn, TABLE_TOUR_COMPARED, "tourRecordingTime") == false) { //$NON-NLS-1$
+
+// SET_FORMATTING_OFF
+
+            // Add new columns
+            SQL.AddCol_Float(stmt, TABLE_TOUR_COMPARED, "tourRecordingTime", DEFAULT_0); //$NON-NLS-1$
```

Comment:
This field is an integer value and not a float

Reply:
For version 19.10, I can only send an e-mail to the translators to do the translation, when the tour database is final.

Reply:
I will correct

Reply:
Fixed, see my [PR](https://github.com/wolfgang-ch/mytourbook/pull/153/)